meta-mentor-staging/recipes-graphics/mesa/mesa-demos_8.2.0.bbappend

```diff
@@ -1,3 +1,6 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI_append = " file://0010-sharedtex_mt-fix-rendering-thread-hang.patch \
-				   file://0001-drop-demos-dependant-on-obsolete-MESA_screen_surface.patch"
+				   file://0001-drop-demos-dependant-on-obsolete-MESA_screen_surface.patch \
+                   file://0002-only-build-GLX-demos-if-needed.patch"
+PACKAGECONFIG[glx] = "--enable-glx-demos,--disable-glx-demos"
+PACKAGECONFIG_append = " glx"
```

Comment:
Minor, but there's no need to use a lazy _append operation here, just use += and remove the leading space.